vkconfig_core/test/override_settings_2_2_2_schema_1_2_1.txt

```diff
@@ -1,26 +1,147 @@
 
 # VK_LAYER_LUNARG_reference_1_2_1
+
```

Comment:
I'm not sure how to interpret this whole file... the descriptions are not helpful, but maybe they're not required to be?

Maybe this file shouldn't change at all?

Reply:
This file was overwritten when I was running some tests, then I inadvertently committed it. I'll back out the change.

Reply:
This file is compared to the output of a test. Since settings files were changed, the expected output is also changed.